kolibri/core/mixins.py

```diff
@@ -28,22 +30,33 @@ class BulkDeleteMixin(object):
 
     # Taken from https://github.com/miki725/django-rest-framework-bulk
 
-    def allow_bulk_destroy(self, qs, filtered):
+    def allow_bulk_destroy(self, qs):
```

Comment:
Ah, so because we're not even looking at the filtered queryset this doesn't matter.

I wonder, could we just call `qs = self.get_queryset()` within this function and avoid having to pass it as an argument at all?

Reply:
Yes, that makes sense